e2e/testdata/repo-sync-helm-cm-update/configmap-1.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Google LLC
```

Comment:
I would advise declaring resources inline in the test as the default choice. It generally makes tests easier to read (no need to go find the files where the resources are actually declared). Our go-based k8s client also applies test labels for cleanup, generally has a better UX, and generally has better error reporting compared to shelling out to kubectl

I think declaring test resources as files should be more of a special case for more complex circumstances

Reply:
Done